src/containers/UserEdit.jsx

```diff
@@ -23,40 +23,93 @@ class UserEdit extends React.Component {
   };
 
   async componentWillMount() {
-    await this.props.mutations.editUser(null)
+    if (!this.props.authType === 'login' && !this.props.authType === 'signup') {
```

Comment:
Maybe just `if (!this.props.authType)` is sufficient (will it be null in contexts where they are already logged in)?